meta-refkit/recipes-computervision/librealsense/librealsense_1.12.0.bbappend

```diff
@@ -7,3 +9,8 @@ EXTRA_OECMAKE = " \
  
 PACKAGES = "${PN} ${PN}-dbg ${PN}-dev ${@bb.utils.contains('DISTRO_FEATURES', 'x11 opengl', '${PN}-examples ${PN}-graphical-examples', '', d)}"
 
+SRC_URI_append = " \
+    file://0001-scripts-removed-bashisms.patch \
```

Comment:
You're right... This should work even with a wildcard version match after the version update. Changing the patch set.